qutebrowser/browser/commands.py

```diff
@@ -1307,6 +1307,32 @@ def on_editing_finished(self, elem, text):
         except webelem.IsNullError:
             raise cmdexc.CommandError("Element vanished while editing!")
 
+    @cmdutils.register(instance='command-dispatcher',
+                       modes=[KeyMode.insert], hide=True, scope='window')
+    def paste_primary(self):
+        """Paste the primary selection at cursor position into the curently
+        selected form field.
+        """
```

Comment:
Docstrings should be formatted like this:

```
"""Short one-line description

(optional) more detailed description
"""
```

In this case, I think `"""Paste the primary selection at the cursor position."""` or so would suffice.